SEED-platform/seed
# 3393 derived col 2 decimal pt hardcode

#### Any background context you want to provide?
Derived Column evaluations came back with any range of decimal points.

#### What's this PR do?
Adds a hardcoded default of 2 decimal points

`NOTE: This is a temporary but solid fix for the demo. Adding a decimal count setting to the derived column editor for each derived column is currently in development.`

#### How should this be manually tested?
- create a derived column that will evaluate to values with more than 2 decimal points. 
- confirm there are only 2 decimal points

#### What are the relevant tickets?
#3393 



#### Screenshots (if appropriate)
dc2 = PM Property ID / 7
dc3 = dc2 / 7
![Screen Shot 2022-07-18 at 10 15 32 AM](https://user-images.githubusercontent.com/58446472/179556115-9161e0dc-258d-49fd-9116-11e03c1dfdba.png)
